### PR TITLE
monero-wallet-rpc - Changed default cryptonote port

### PR DESCRIPTION
**Pull request by SyntheticBird45**
- Removed `localhost:8080` default value to address string in wallet rpc
- If `set_daemon` rpc address string is empty, mwr will return an error and explicit message

### CODE REVIEW

**Comment by SyntheticBird45** [comment]
ready

File: src/wallet/wallet_rpc_server.cpp [hunk 1]
```diff
@@ -34,6 +34,7 @@
 #include <boost/preprocessor/stringize.hpp>
 #include <cstdint>
 #include "include_base_utils.h"
+#include "wallet_rpc_server_error_codes.h"
```
Comment (vtnerd) [inline src/wallet/wallet_rpc_server.cpp:37]: Why this change?

Comment (SyntheticBird45) [reply src/wallet/wallet_rpc_server.cpp:37]: first commit i added an error message and the WRONG_ADDRESS core rpc error. Clangd automatically included its header. forgot to remove it

**Review by SChernykh — APPROVED** [review]

**Review by SChernykh — APPROVED** [review]

**Review by rbrunner7 — APPROVED** [review]
